cmd/truss/main.go

```diff
@@ -29,8 +30,32 @@ var (
 	verboseFlag    = flag.Bool("v", false, "Verbose output with stack traces for errors.")
 )
 
+const (
+	noVersion   string = "<no-version>"
+	noBuildDate string = "<no-build-date>"
+)
+
+var (
+	Version   string = noVersion
+	BuildDate string = noBuildDate
+)
+
 func init() {
+	buildinfo := new(bytes.Buffer)
+	if Version != "" && Version != noVersion {
+		buildinfo.WriteString("version: ")
+		buildinfo.WriteString(Version)
+		buildinfo.WriteByte(' ')
+	}
+	if BuildDate != "" && BuildDate != noBuildDate {
+		buildinfo.WriteString("built: ")
+		buildinfo.WriteString(strings.Replace(BuildDate, "_", " ", -1))
+	}
+
 	flag.Usage = func() {
+		if buildinfo.Len() > 0 {
+			fmt.Fprintf(os.Stderr, "Truss (%s)\n", strings.TrimSpace(buildinfo.String()))
```

Comment:
In my mind, I was perceiving this as a product banner, where the product is "Truss," which doesn't depend on the name of the binary. But it doesn't matter too much to me. I mean, realistically we're not going to rename `cmd/truss`, so it's all the same, and lowercase is fine. I'll update since I have to fix what I messed up in my "simplify" commit.